src/Sulu/Bundle/MediaBundle/Resources/js/containers/MultiMediaDropzone/MultiMediaDropzone.js

```diff
@@ -178,7 +178,7 @@ class MultiMediaDropzone extends React.Component<Props> {
                 <SingleListOverlay
                     adapter="column_list"
                     clearSelectionOnClose={true}
-                    itemDisabledCondition="!!locked"
+                    itemDisabledCondition="!!locked || (_permissions && !_permissions.add)"
```

Comment:
we check the permissions in a similar manner in some other places, including the toolbar actions created in the `PageAdmin`: https://github.com/sulu/sulu/blob/b6613b73d94580d0c5daf65dd6c9f719804458f3/src/Sulu/Bundle/PageBundle/Admin/PageAdmin.php#L122

Reply:
Where does `!!locked` come from I don't see any definition of it?

Reply:
looks like the `CollectionController` does not use our `ListBuilder`. the `locked` property is defined here:  https://github.com/sulu/sulu/blob/e3244a8fe56379d02504e47c4c31b2681223e4e1/src/Sulu/Bundle/MediaBundle/Api/Collection.php#L510-L520